client/sites-dashboard/components/sites-table-filter-tabs.tsx

```diff
@@ -8,25 +10,70 @@ interface SitesTableFilterTabsProps {
 	className?: string;
 }
 
+interface FilteredSites {
+	[ name: string ]: SiteData[];
+}
+
+interface SiteTab extends Omit< TabPanel.Tab, 'title' > {
+	title: React.ReactChild;
+}
+
+const SitesTabPanel = styled( TabPanel )`
+	.components-tab-panel__tabs {
+		overflow-x: auto;
+		padding-bottom: 10px;
+	}
+
+	.components-tab-panel__tabs-item {
+		--wp-admin-theme-color: var( --studio-gray-100 );
+		color: var( --studio-gray-60 );
+		padding: 0;
+		margin-right: 24px;
+		font-size: 16px;
+		flex-shrink: 0;
+	}
+	.components-tab-panel__tabs-item:hover,
+	.components-tab-panel__tabs-item.is-active,
+	.components-tab-panel__tabs-item.is-active:focus,
+	.components-tab-panel__tabs-item:focus:not( :disabled ) {
+		box-shadow: inset 0 -2px 0 0 var( --wp-admin-theme-color );
+		color: var( --studio-gray-100 );
+	}
+`;
+
 export function SitesTableFilterTabs( {
 	allSites,
 	children: renderContents,
 	className,
 }: SitesTableFilterTabsProps ) {
 	const { __ } = useI18n();
 
+	let tabs: SiteTab[] = [
+		{ name: 'all', title: __( 'All' ) },
+		{ name: 'launched', title: __( 'Launched' ) },
+		{ name: 'private', title: __( 'Private' ) },
+		{ name: 'coming-soon', title: __( 'Coming soon' ) },
+	];
```

Comment:
Should we write the filter state to a query param, and then restore the state from query param on page load?

Feel free to capture this as a follow-up issue too.

Reply:
That's a good idea. I'll create an issue and put it in the backlog so we work on that when we have the bandwidth.